### PR TITLE
[WD-154] responsive navbar

### PR DESCRIPTION
**Pull request by Edouarddm**
Capture d’écran 2023-11-24 à 17 41 41

### CODE REVIEW

File: app/assets/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -36,3 +36,21 @@
...
   }
 }
+
+@media (min-width: 100px) and (max-width: 575px) {
+  .navbar-collapse {
+    background: white;
+    border: solid 1px lightgrey;
+    border-radius: 12px;
+    padding: 0.5em;
+    align-items: flex-end;
```
Comment (christinasugiono) [inline app/assets/stylesheets/components/_navbar.scss:46]:
```suggestion
    margin-left: auto;
    flex-basis: auto;
    // align-items: flex-end;
```

you can try this to make the navbar not full width :D

Screenshot 2023-11-28 at 11 50 22

Comment (Edouarddm) [reply app/assets/stylesheets/components/_navbar.scss:46]: I was a little bit struggling with this, thanks!

Comment (Edouarddm) [reply app/assets/stylesheets/components/_navbar.scss:46]: Well, at the end even with the help of Daniel we couldn't solve this. With your code it gives this !
Capture d’écran 2023-11-28 à 15 50 37

Comment (christinasugiono) [reply app/assets/stylesheets/components/_navbar.scss:46]: Oh nooo :( Let's go with the first one first then

**Review by christinasugiono — APPROVED** [review]

**Pull request merged**